docs/PerformAudioPassThru/index.md

```diff
@@ -1,16 +1,20 @@
 ## PerformAudioPassThru
 
-`PerformAudioPassThru` feeds you with audio data through the vehicle’s
-microphone. The audio data could be used in cloud-based and on-line voice
-recognition to achieve dynamic and diversified user interaction, like POI search, information query. Or even record the voice when driver is singing.
-The audio data is in form of PCM, and its sampling rate, bit width, timeout can be set in the `PerformAudioPassThru` request per your requirement. The headunit transmits its supported parameters via the `registerAppInterface` response. 16bit/16kHz is a widely supported configuration, which delivers good audio quality.
+`PerformAudioPassThru` feeds you with audio data through the vehicle’s microphone. The audio data could be used in cloud-based and on-line voice recognition to achieve dynamic and diversified user interaction, like POI search, information query, or even record the voice when the driver is singing. The audio data is in form of PCM, and its sampling rate, bit width, timeout can be set in the `PerformAudioPassThru` request per your requirement. The headunit transmits its supported parameters via the `registerAppInterface` response. 16bit/16kHz is a widely supported configuration, which delivers good audio quality.
```

Comment:
> `PerformAudioPassThru` feeds you with audio data through the vehicle’s microphone.

remove "with", change "through" to "from"

> The audio data could be used in cloud-based and on-line voice recognition to achieve dynamic and diversified user interaction, like POI search, information query, or even record the voice when the driver is singing.

Change "could" to "can". Remove "and diversified". Change "interaction, like" to "interaction, such as". Remove "the voice".

> The audio data is in form of PCM, and its sampling rate, bit width, timeout can be set in the `PerformAudioPassThru` request per your requirement. The headunit transmits its supported parameters via the `registerAppInterface` response. 16bit/16kHz is a widely supported configuration, which delivers good audio quality.

"The Audio data will be in PCM format. The sample rate, bit width, and timeout can be set, however, the supported parameters will be sent in the `registerAppInterface` response. Generally, 16 bit width, 16KHz sample rate will be supported.